File: an toan phan mem/Baocao_couiky/backendBanhang/TodoAPI/Models/GIOHANG.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace BANHANG.Models
{
    [Table("GIOHANG")]
    public class GIOHANG
    {
        [Key]
        [Column("IDGIOHANG")]
        public long IDGIOHANG { get; set; }

        [Column("MAKH")]
        public long MAKH { get; set;}
       
        [Column("NGAYMUA")]
        public DateTime NGAYMUA { get; set; }

        [Column("TTTHANHTOAN")]
        public bool TTTHANHTOAN { get; set; }

        [Column("HINHTHUCTT")]
        public string HINHTHUCTT { get; set; }

        [Column("TONGTIENGH")]
        public long TONGTIENGH { get; set; }

        [Column("TONGTHANHTOAN")]
        public long TONGTHANHTOAN { get; set; }

        [ForeignKey("MAKH")]
        public virtual KHACHHANG khachhang { get; set; }
    }
}

File: an toan phan mem/front-end/atpm_cuoiky-master/FE/src/app/model/giohang.ts
export class GioHang {
    IDGIOHANG : number;
    MAKH : number;
    NGAYMUA : Date;
    TTTHANHTOAN : boolean;
    HINHTHUCTT : string;
    TONGTIENGH : number;
    TONGTHANHTOAN : number;
    constructor(
        MAKH : number,
        NGAYMUA : Date,
        TTTHANHTOAN : boolean,
        HINHTHUCTT : string,
        TONGTIENGH : number,
        TONGTHANHTOAN : number){
        // this.IDGIOHANG = IDGIOHANG;
        this.MAKH = MAKH;
        this.NGAYMUA = NGAYMUA;
        this.TTTHANHTOAN = TTTHANHTOAN;
        this.HINHTHUCTT = HINHTHUCTT;
        this.TONGTIENGH = TONGTIENGH;
        this.TONGTHANHTOAN = TONGTHANHTOAN;
    }
}